dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1520,7 +1539,33 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 			return;
 		}
 
-		GameMessages::SendVehicleAddPassiveBoostAction(vehicle->GetObjectID(), UNASSIGNED_SYSTEM_ADDRESS);
+		if (args.size() >= 1) {
+			float time;
+
+			if (!GeneralUtils::TryParse(args[0], time)) {
+				ChatPackets::SendSystemMessage(sysAddr, u"Invalid boost time.");
+				return;
+			} else {
+				GameMessages::SendVehicleAddPassiveBoostAction(vehicle->GetObjectID(), UNASSIGNED_SYSTEM_ADDRESS);
+				entity->AddCallbackTimer(time, [vehicle]() {
+					if (!vehicle) return;
+					GameMessages::SendVehicleRemovePassiveBoostAction(vehicle->GetObjectID(), UNASSIGNED_SYSTEM_ADDRESS);
+					});
+			}
+		} else {
+			GameMessages::SendVehicleAddPassiveBoostAction(vehicle->GetObjectID(), UNASSIGNED_SYSTEM_ADDRESS);
+		}
+
+	}
```

Comment:
Feedback for invalid number of arguments should be added here

Reply:
it doesn't take any args? so given more args nothing will break and it won't be invalid